projects/syzkaller/Dockerfile

```diff
@@ -17,11 +17,11 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER mmoroz@chromium.org
 
-RUN go get -u -d github.com/google/syzkaller/...
+RUN go get -u -d github.com/google/syzkaller/prog
```

Comment:
sorry I can't debug myself (see go/mmoroz-ooo), but my first guess would be that this line isn't doing what we expect it to do

you can also try this locally, check out OSS-Fuzz repo and follow https://google.github.io/oss-fuzz/getting-started/new-project-guide/#prerequisites, then do:

```
python infra/helper.py build_image syzkaller
python infra/helper.py shell syzkaller
```

that would open bash inside syzkaller build image, i.e. you can explore the file system interactively and/or try checking out the sources some other way